b2sdk/exception.py

```diff
@@ -391,7 +391,7 @@ def __init__(self, limit, exception_info_list):
 
     def __str__(self):
         exceptions = '\n'.join(str(wrapped_error) for wrapped_error in self.exception_info_list)
-        return 'FAILED to upload after {} tries. Encountered exceptions: {}'.format(
+        return 'FAILED to upload after {} seconds of trying. Encountered exceptions: {}'.format(
             self.limit,
```

Comment:
this is a public member of the class and it changed meaning - doing that would require a major version bump that we generally don't want to do